docs/contribute/docker-release-workflow.md

```diff
@@ -0,0 +1,66 @@
+# Docker Hub Release Workflow (`release.yml`)
+
+> **File location:** `.github/workflows/release.yml`  
+> **Introduced in:** PR [#271](https://github.com/Lind-Project/lind-wasm/pull/271)
```

Comment:
taste nit: I find this preamble a bit cluttering. The PR number isn't really relevant for the purpose of this doc, and could also be inferred from the git history. And the file location of the workflow could be shown more subtly in the opening paragraph. See suggestion below